WalletWasabi/WabiSabi/Backend/WabiSabiConfig.cs

```diff
@@ -136,4 +158,22 @@ public void MakeNextCoordinatorScriptDirty()
 			ToFile();
 		}
 	}
+
+	private static ImmutableSortedSet<ScriptType> GetScriptTypes(bool P2wpkh, bool P2tr)
```

Comment:
Please use camelCase for parameters as follow:

```suggestion
	private static ImmutableSortedSet<ScriptType> GetScriptTypes(bool p2wpkh, bool p2tr)
```

Reply:
Fixed in https://github.com/zkSNACKs/WalletWasabi/pull/8831/commits/6816f2d064a441f0dfa38e5ee2b268aa2916dc1c  and https://github.com/zkSNACKs/WalletWasabi/pull/8831/commits/ac8c5168e817da8d3ac79fa3362b355dada6cf70.